tests/cpp/operator/mkldnn.cc

```diff
@@ -803,25 +813,36 @@ TEST(MKLDNN_BASE, MKLDNNSum) {
   for (int i = 0; i < in_arrs.size(); i++) {
     auto in_arr = in_arrs[i];
     auto in_arr2 = in_arrs2[i];
-    std::vector<NDArrayAttrs> out_arrs = GetTestOutputArrays(in_arr.arr.shape(), pds);
-    if (!SupportMKLDNN(in_arr.arr) || !in_arr.arr.IsMKLDNNData() || in_arr.arr.IsView())
+    if (!SupportMKLDNN(in_arr.arr))
       continue;
-
+    if (in_arr.arr.IsMKLDNNData() && in_arr.arr.IsView()) {
+      in_arr.arr = in_arr.arr.Reorder2Default();
+      in_arr2.arr = in_arr2.arr.Reorder2Default();
+    }
```

Comment:
do we really need to have this? this is test code. if we know that a function that we test doesn't support this case, we can simply skip it. i wonder what benefit we can get from enabling this test case.
The input arrays contain the default format, so this test case has been covered.